dora/core/client/fs/src/main/java/alluxio/client/file/dora/WorkerLocationPolicy.java

```diff
@@ -63,9 +68,12 @@ private Factory() {} // prevent instantiation
      */
     public static WorkerLocationPolicy create(AlluxioConfiguration conf) {
       try {
-        return CommonUtils.createNewClassInstance(
+        WorkerLocationPolicy workerLocationPolicy = CommonUtils.createNewClassInstance(
             conf.getClass(PropertyKey.USER_WORKER_SELECTION_POLICY),
             new Class[] {AlluxioConfiguration.class}, new Object[] {conf});
+        LOG.info(String.format("Using worker location policy: %s",
```

Comment:
make this DEBUG. This log will show up too many times.

Reply:
and when you test, you can modify log4j.properties like 
https://docs.alluxio.io/os/user/stable/en/administration/Basic-Logging.html#configuring-log-levels